src/main/java/com/blackduck/integration/blackduck/codelocation/signaturescanner/command/ToolsApiScannerInstaller.java

```diff
@@ -140,8 +144,14 @@ protected HttpUrl getDownloadUrl() throws BlackDuckIntegrationException {
             platform = MAC_PLATFORM_PARAMETER_VALUE;
         } else if (OperatingSystemType.WINDOWS == operatingSystemType) {
             platform = WINDOWS_PLATFORM_PARAMETER_VALUE;
-        } else {
+        } else if (OperatingSystemType.LINUX == operatingSystemType && !isAlpineLinux()) {
```

Comment:
This is likely personal preference but I kind of like checking if it IS Alpine Linux and specifically setting the platform to Alpine if so. Then the else could be the fallback where you set the platform to general linux if not. Reads a bit clearer to me but I'm really fine either way.